src/app/modules/budget/budget-overview/budget-overview.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
jak to z tego brancha budżetu to spoko by bylo powywalać te plików testów bo są totalnie aktualnie niepotrzebne a tylko śmietnik robią

Reply:
można ale to wywale w branchu budget jak poleci PR

Reply:
usunięte